### ISSUE TITLE
[RFE]Is there a way to pull the redhat container catalog images from UI (Not using oc client)?

### ISSUE THREAD

**Issue by anshulbehl** [openshift/origin#19736]
on the link below

https://access.redhat.com/containers/?tab=images&platform=openshift#/registry.access.redhat.com/rhcertification/redhat-certification-management

It says:

Container images that are made to run with OpenShift platforms can either be pulled from the command line (using the oc command) or from the OpenShift Dashboard GUI interface. The following is one command line you could use to pull the rhcertification/redhat-certification-management container image to the local system for use with OpenShift:

I am concerned with the statement

> or from the OpenShift Dashboard GUI interface

Just want to know is there a way to do this yet?

**Comment by jwforres** [comment]
We have a "Deploy Image" action but that is if you intend to run the image.  Are you asking about a GUI for setting up a new image stream?

@openshift/sig-user-interface

**Comment by anshulbehl** [comment]
@jwforres  That is correct, is there a way in UI to add a new image stream. I have also filed an RH bugzilla for the same and letting you know the ID just so we don't end up doing duplicate efforts.

https://bugzilla.redhat.com/show_bug.cgi?id=1572068

For the suggestion of the enhancement here, additions of image streams in UI for registries of Docker and RHCC(Container Catalog) should prove beneficial.

**Where this is coming from:**
I have created an openshift supported image for the RH partners that my team supports, which is released to RHCC. In order for partners to use the image the image stream has to be added from oc utility which is also sort of the only CLI task need to run the image on openshift online if a partner has an account there. Just looking for solutions to make this easy for them, and minimal usage of oc utility.
